styles/style.css

```diff
@@ -161,10 +190,34 @@ p {
 
 /* Article */
 
+/* Articles row */
+.flexrow{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
 .article {
   margin-bottom: 1rem;
+  width: 300px;
+  border: 1px solid var(--grey-light);
+  border-radius: 12px;
+  padding: 10px;
+  margin: 5px;
+} 
+
+/* following codes used on the class to inline image and text div of those articles on sidebar display in columns, align
+items:center used to just the image on center of whole size of that article */
+.inArt{
```

Comment:
Perhaps a better name for this would be `in-article`.

Generally we want to avoid abbreviating in variable names. Try to imagine looking at this code in a year and try to name it so that it would still be clear what the variable does. You can read more here: https://syllabus.codeyourfuture.io/guides/code-style-guide#avoid-short-names

Reply:
changed this along with some other correction that you suggested abouve